journal/src/main/scala/com/evolutiongaming/kafka/journal/Journal.scala

```diff
@@ -317,7 +317,28 @@ object Journal {
 
     val default: CallTimeThresholds = CallTimeThresholds()
 
-    implicit val configReaderCallTimeThresholds: ConfigReader[CallTimeThresholds] = deriveReader[CallTimeThresholds]
+    implicit val configReaderCallTimeThresholds: ConfigReader[CallTimeThresholds] = new ConfigReader[CallTimeThresholds] {
+      val fdReader = implicitly[ConfigReader[FiniteDuration]]
+      override def from(cur: ConfigCursor): ConfigReader.Result[CallTimeThresholds] = {
+        for {
+          objCur <- cur.asObjectCursor
+          appendCur = objCur.atKeyOrUndefined("append")
+          append <- if (appendCur.isUndefined) Right(CallTimeThresholds().append) else fdReader.from(appendCur)
+
+          readCur = objCur.atKeyOrUndefined("read")
+          read <- if (readCur.isUndefined) Right(CallTimeThresholds().read) else fdReader.from(readCur)
+
+          pointerCur = objCur.atKeyOrUndefined("pointer")
+          pointer <- if (pointerCur.isUndefined) Right(CallTimeThresholds().pointer) else fdReader.from(pointerCur)
+
+          deleteCur = objCur.atKeyOrUndefined("delete")
+          delete <- if (deleteCur.isUndefined) Right(CallTimeThresholds().delete) else fdReader.from(deleteCur)
+
+          purgeCur <- objCur.atKeyOrUndefined("purge")
+          purge <- if (purgeCur.isUndefined) Right(CallTimeThresholds().purge) else fdReader.from(purgeCur)
+        } yield CallTimeThresholds(append, read, pointer, delete, purge)
```

Comment:
please elaborate why this is required

Reply:
that's the DRAFT part. I'm not sure how to proceed with the config codec derivation since cross-compilation doesn't work with macros. This one is here for now to get the subproject to compile